hedera-mirror-rest-java/src/main/java/com/hedera/mirror/restjava/common/SpringApplicationContext.java

```diff
@@ -0,0 +1,25 @@
+package com.hedera.mirror.restjava.common;
+
+import org.springframework.beans.BeansException;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.ApplicationContextAware;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SpringApplicationContext implements ApplicationContextAware {
+
+    private static ApplicationContext context;
+
+    public static <T extends Object> T getBean(Class<T> beanClass) {
+        return context.getBean(beanClass);
+    }
+
+    @Override
+    public void setApplicationContext(ApplicationContext context) throws BeansException {
+        setContext(context);
+    }
+
+    private static synchronized void setContext(ApplicationContext context) {
+        SpringApplicationContext.context = context;
+    }
```

Comment:
nit: Could use a static final AtomicReference.

```suggestion
       CONTEXT.set(context);
    }
```